account_payment_sale/views/sale_order_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2014-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+
+<record id="view_order_form" model="ir.ui.view">
+    <field name="name">account_payment_sale.sale_order.form</field>
+    <field name="model">sale.order</field>
+    <field name="inherit_id" ref="sale.view_order_form"/>
+    <field name="arch" type="xml">
+        <field name="payment_term_id" position="after">
+            <field name="payment_mode_id"
+                domain="[('payment_type', '=', 'inbound')]"
+                widget="selection"/>
```

Comment:
Can change this widget by:
```suggestion
                options="{'no_open': True, 'no_create': True}"
```
I suggest this change why we have a lot of client what have a lot payment modes and the selection widget only show part of them.